web/discord_relay/queue.go

```diff
@@ -50,6 +51,51 @@ func sendMessage(discord *discordgo.Session, message MessageStruct) {
 	discord.WebhookExecute(WebhookId, WebhookSecret, true, params)
 }
 
+func sendConnect(discord *discordgo.Session, message MessageStruct) {
+	var contentBuilder strings.Builder
+	contentBuilder.WriteString(message.Content)
+
+	params := &discordgo.WebhookParams{
+		AllowedMentions: &discordgo.MessageAllowedMentions{
+			Parse: []discordgo.AllowedMentionType{},
+		},
+		Content:   contentBuilder.String(),
+		Username:  "Relay",
+	}
+
+	discord.WebhookExecute(WebhookId, WebhookSecret, true, params)
+}
+
+func sendConnectMessage(discord *discordgo.Session, message MessageStruct) {
+	var contentBuilder strings.Builder
+	contentBuilder.WriteString(message.Content)
+
+	params := &discordgo.WebhookParams{
+		AllowedMentions: &discordgo.MessageAllowedMentions{
+			Parse: []discordgo.AllowedMentionType{},
+		},
+		Content:   message.SteamName + " | " + message.SteamId + " has connected to the server." ,
+		Username:  "Relay",
+	}
+
+	discord.WebhookExecute(WebhookId, WebhookSecret, true, params)
+}
+
+func sendDisconnectMessage(discord *discordgo.Session, message MessageStruct) {
+	var contentBuilder strings.Builder
```

Comment:
string.Builder is useless if you only write one thing. it also looks like you arent using contentBuilder in this function
go code will not compile if you declare a variable then dont use it